src/connectedk8s/setup.py

```diff
@@ -17,7 +17,7 @@
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 
-VERSION = '0.2.7'
+VERSION = '0.2.8'
```

Comment:
revert this change. We will be raising another PR with the increase in version

Reply:
Done